optuna/trial.py

```diff
@@ -453,7 +453,9 @@ def set_system_attr(self, key, value):
     def _suggest(self, name, distribution):
         # type: (str, BaseDistribution) -> Any
 
-        if self._is_relative_param(name, distribution):
+        if self._is_manual_param(name, distribution):
+            param_value = self.system_attrs['_manual_params'][name]
```

Comment:
Although this was originally named by me, I think `_manual_params` isn't appropriate as the key name.
How about changing it to `_fixed_params` or another nice name?

Reply:
Thank you for your review!

Okay. I'll rename to `_fixed_params`. BTW, I've just thought `fixed_params` might be enough because the users basically don't access to `system_attrs`.

Reply:
Thank you!

> `fixed_params` might be enough 

I agree with you. I think it's okay to remove `_` prefix from the name.